wsclient/src/main/java/org/hpccsystems/ws/client/antlr/EclRecordReader.java

```diff
@@ -396,6 +396,30 @@ else if (currentrec != null)
         }
     }
 
+    /**
+     * {@inheritDoc}
+     *
+     * <p>
+     * When entering a DEFAULT option for Set Of, set the columnvalue of the current field/rec to its value
+     * </p>
+     */
+    @Override
+    public void enterSetdefaultval(EclRecordParser.SetdefaultvalContext ctx)
+    {
+        String val = ctx.getChild(2).getText();
+        val = val.replace("['", "");
```

Comment:
What happens if the string contains a square bracket?
e.g.  DEFAULT(['[]').

Reply:
well ['[]'] will not match [' or '] so you would end up with a default value of [ -- we could shift it to replace first and replace last this would only be an issue with DEFAULT(['['']) which doesn't make sense as you would have to escape the ' and it would need to be DEFAULT(['[\'']) and [\' wouldn't match the replacement